dogs.json

```diff
@@ -32,5 +32,11 @@
     { "dog": "roy", "img": "http://wpc.3347.edgecastcdn.net/003347/dogs/roy1.jpg"},
     { "dog": "wicket", "img": "http://wpc.3347.edgecastcdn.net/003347/dogs/wicket1.jpg"},
     { "dog": "biggie", "img": "http://wpc.3347.edgecastcdn.net/003347/dogs/biggie_pen.jpg"},
-    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_summer_time_livin.jpg"}
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_summer_time_livin.jpg"},
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_at_Amber.jpg"},
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker.JPG"},
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_3.jpg"},
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_nappy.jpg"},
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_photogenic.JPG"},
+    { "dog": "tucker", "img": "http://wpc.E37B1.taucdn.net/00E37B1/dogs/tucker_watch_dog.jpg"},
```

Comment:
Can't end with a trailing comma. This is invalid Javascript.

This ain't perl!

Reply:
Javascript 1, Meza 0